docs/sources/alerting/fundamentals/notifications/notification-policies.md

```diff
@@ -18,130 +18,93 @@ labels:
-
-To inherit a contact point from the parent policy, leave it blank. To override the inherited grouping options, enable **Override grouping**. To override the inherited timing options, enable **Override general timings**.
-
-### Inheritance example
-
-The example below shows how the notification policy tree from the previous example allows the child policies of the `team=operations` to inherit its contact point.
-
-In this way, you can avoid having to specify the same contact point multiple times for each child policy.
-
-{{< figure src="/media/docs/alerting/notification-inheritance.png" max-width="750px" caption="Notification policy inheritance" >}}
-
-## Additional configuration options
-
-### Grouping
-
-Grouping is an important feature of Grafana Alerting as it allows you to batch relevant alerts together into a smaller number of notifications. This is particularly important if notifications are delivered to first-responders, such as engineers on-call, where receiving lots of notifications in a short period of time can be overwhelming and in some cases can negatively impact a first-responders ability to respond to an incident. For example, consider a large outage where many of your systems are down. In this case, grouping can be the difference between receiving 1 phone call and 100 phone calls.
-
-Choose how alerts are grouped together using the Group by option in a notification policy. By default, notification policies in Grafana group alerts together by alert rule using the `alertname` and `grafana_folder` labels (since alert names are not unique across multiple folders). If you want to group alerts by something other than the alert rule, change the grouping to any other combination of labels.
-
-#### Disable grouping
-
-If you want to receive every alert as a separate notification, you can do so by grouping by a special label called `...`. This is useful when your alerts are being delivered to an automated system instead of a first-responder.
-
-#### A single group for all alerts
-
-If you want to receive all alerts together in a single notification, you can do so by leaving Group by empty.
-
-### Timing options
-
-The timing options decide how often notifications are sent for each group of alerts. There are three timers that you need to know about: Group wait, Group interval, and Repeat interval.
-
-#### Group wait
+## Inheritance
 
-Group wait is the amount of time Grafana waits before sending the first notification for a new group of alerts. The longer Group wait is the more time you have for other alerts to arrive. The shorter Group wait is the earlier the first notification is sent, but at the risk of sending incomplete notifications. You should always choose a Group wait that makes the most sense for your use case.
+In addition to child policies being a useful concept for routing alert instances, they also inherit properties from their parent policy. This also applies to child policies of the default notification policy.
```

Comment:
here we talk about child again...so same thing as above really

Reply:
https://github.com/grafana/grafana/pull/88656/commits/7d62f0bdabf263a9fc16993e384aec1f9cccffaa tries to clarify about sibling and child policies